doc/manual/generate-manpage.nix

```
@@ -116,9 +116,11 @@ let
             storeInfo = commandInfo.stores;
             inherit inlineHTML;
           };
+          hasInfix = infix: content:
+            builtins.stringLength content != builtins.stringLength (replaceStrings [ infix ] [ "" ] content);
```

Let's leave a breadcrumb for when the `git blame` doesn't point to the commit anymore.

```suggestion
            # An alternate implementation with builtins.match stack overflowed on some systems.
            # See https://github.com/NixOS/nix/pull/11090 or commit c5879a36f59
            builtins.stringLength content != builtins.stringLength (replaceStrings [ infix ] [ "" ] content);
```

---

Oh it doesn't let me push to your branch. Could you apply the suggestion?